packages/monitors/index.js

```diff
@@ -103,7 +103,11 @@ async function run({
      * Set variables common to all monitors
      *
      ***************************************/
-    const [networkId, latestBlock] = await Promise.all([web3.eth.net.getId(), web3.eth.getBlock("latest")]);
+    const [networkId, latestBlock, chainId] = await Promise.all([
```

Comment:
what is the difference between `net.getId` and `getChainId`? I tested this and they look like they are always the same number.

Reply:
They are usually the same number, but not always https://besu.hyperledger.org/en/stable/Concepts/NetworkID-And-ChainID/#:~:text=Usually%20the%20network%20ID%20is,the%20%2D%2Dnetwork%2Did%20option. For all things outside of node consensus, chainId is recommended over networkId.

Reply:
do we operate on any networks that dont have these matching?

Reply:
Nope, I don't think so. Although I think this can differ in local hardhat forks if I remember correctly.

Reply:
My basic push is that we should avoid expanding the use of networkId as it isn't recommended for applications.